README.md

```diff
@@ -1,60 +1,58 @@
+This project builds packages and images for the NI Linux RT distribution.
 
-Build Steps
-------
+NI Linux RT is a
+[Real-Time-scheduling](https://rt.wiki.kernel.org/index.php/CONFIG_PREEMPT_RT_Patch)
+enabled OS, for use on NI's embedded hardware devices. This project uses the
+[OpenEmbedded](http://www.openembedded.org/wiki/Main_Page) framework to build
+and package the kernel, software, images, and installation media which compose
+NI Linux RT.
 
-1. Get the source by running the following commands:
-
-        git clone https://github.com/ni/nilrt.git
-        cd nilrt
-        git checkout nilrt/<release>
-        git submodule init
-        git submodule update --remote --checkout
-
-2. Set up your shell environment to run bitbake:
-
-    Source NILRT's environment script by running the following command:
-
-        . ni-oe-init-build-env
-
-    This will setup the needed environment variables and build
-configuration files for building through OpenEmbedded build system.
-Note that the configuration files (that exist in the build/conf
-directory) include some basic default configurations, allowing
-modification or overriding of these default configurations.
+Community (non-NI) developers can use this project to build packages for
+open-source software projects, which are installable to their existing NI Linux
+RT devices - including custom Linux kernels and kernel modules.
 
-    Set an appropriate MACHINE variable so that bitbake can tune builds
-of NILRT for your hardware.
 
-    Run the following command to configure for ARM targets:
-
-        export MACHINE=xilinx-zynqhf
-
-    or the following command to configure for x64 targets:
-
-        export MACHINE=x64
-
-    **NOTE** It's not recommended to run bitbake for different
-MACHINE's in the same workspace (build directory).
-
-3. Build the package or packages that you want for your target.
-For example, to build Python, Ruby, and Apache for Zynq targets, run the
-following commands:
+## Building Packages
 
-        bitbake python ruby apache2
+1. Get the source by running the following commands:
+    ```bash
+    git clone https://github.com/ni/nilrt.git
+    cd nilrt
+    git checkout nilrt/$release
+    git submodule init
+    git submodule update --remote --checkout
+    ```
+
+2. Set up your shell environment to run bitbake by sourcing the init environment
+file:
+    ```bash
+    . nilrt-build-init.env
+    ```
+
+    This will setup the needed environment variables and build configuration
+    files for building through OpenEmbedded build system. Note that the
+    configuration files (that exist in the build/conf directory) include some
+    basic default configurations, allowing modification or overriding of these
+    default configurations.
+
+3. Build the package or packages that you want for your target. For example, to
+build the Python3, Ruby, and Apache targets, run the following commands:
+
+    ```bash
+    bitbake python3 ruby apache2
+    ```
 
```

Comment:
How about adding instructions to use the `bitbake -s` command? That could be useful when you don't know the full name of the package.

Reply:
That's definitely useful to know, but I don't think it's on the *critical path* that the README should cover. It seems to me to be in the same category as the other useful bitbake debug commands, which are better covered by dedicated bitbake docs, I think.